secator/rich.py

```
@@ -4,13 +4,26 @@
 
 
+class LeftJustifiedHeading(Heading):
+    def __rich_console__(self, console: Console, options: ConsoleOptions) -> RenderResult:
+        yield from console.render(self.text, options=options.update(justify="left"))
+
+
+class CustomMarkdown(Markdown):
+    elements = {
+        **Markdown.elements,
+        "heading_open": LeftJustifiedHeading,
+    }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)